src/component/ButtonPanel.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Buttons from './Button';
+import './ButtonsPanel.css';
+
+const ButtonsPanel = ({ clickHandler }) => (
```

Comment:
- [ ] While your solution works, Microverse requires you to import your clickHandler from props and create a function that should be used in each button by the clickHandler named handleClick, please implement this small change, everything else looks awesome!